package.json

```diff
@@ -49,6 +49,8 @@
 		"cross-env": "5.2.0",
 		"electron": "4.0.5",
 		"electron-builder": "20.38.5",
+		"husky": "^1.3.1",
```

Comment:
Need to use exact versions, rather than matching all minor releases.

Reply:
Done